src/components/Posts/SinglePost.jsx

```diff
@@ -21,7 +22,7 @@ export function SinglePost({ article, id }) {
   useEffect(() => {
     (async function getComments() {
       const articleRef = db.collection("articles").doc(id);
-      const commentsRef = articleRef.collection("comments").orderBy("timestamp", "desc");
+      const commentsRef = articleRef.collection("comments").orderBy("timestamp", "asc");
```

Comment:
descending to ascending? right, that's what you did here?

Reply:
Are you familiar with .json files? I'm writing the resources .json right now. I'm not sure where to put it and if are good with them will you review it when I'm done?

Reply:
> descending to ascending? right, that's what you did here?

Yes



> Are you familiar with .json files? I'm writing the resources .json right now. I'm not sure where to put it and if are good with them will you review it when I'm done?

Sure. I think it's better to create a separate repo for this.

Reply:
Will you continue this conversation on the merge review I just sent you and ronnie

Reply:
Yes. Will reply there.